cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+
+    Channel:
+      type: string
+      enum:
+        - standard
+        - ecommerce
+      examples:
+        - standard
+      description: Channel that was used by the cardholder for executing the transaction, allows the identification of eCommerce transactions.
+
+    TransactionCategory:
+      title: Transaction category
+      type: string
+      maxLength: 50
+      examples:
+        - '100: Cash withdrawal'
+        - '103: ATM operator refund'
+        - '117: P2P payment charge sender'
+        - '118: P2P payment credit recipient'
+        - merchant
+        - fee
+        - atm
+      description: Category of the transaction, defined by the Card Service Provider.
+
+    CardholderReference:
+      title: Cardholder reference
+      type: string
+      maxLength: 50
+      examples:
+        - Peter Schweizer
+      description: Reference to the cardholder who owns the card which was used for the transaction; not provided by all Card Service Providers.
+
+    RelatedTransactionId:
+      title: Transaction reference
+      type: string
+      format: uuid
+      maxLength: 50
+      examples:
+        - 9e19df54-47cb-494b-84dc-1c126e95a05c
+      description: Reference to another transaction or authorization that is linked to the current one, e.g. for separately charged fees, chargeback credits.
+
+    FeeName:
+      title: Fee name
+      type: string
+      maxLength: 50
+      examples:
+        - transactionTipAmount
+        - transactionSurchargeAmount
+      description: Name of the fee that is charged for the transaction; value is defined by the Issuer.
+
+    FeePercentage:
+      title: Fee percentage
+      type: number
+      examples:
+        - 3.5
+      description: >
+        Percentage value of the transactionAmount for the fee type; not provided by all Card Service Providers.
+        Values are provided to be compatible with the % notation. For example a value 3.5 is to be interpreted as 3.5% (= 0.035 as decimal).
+
+    FeeAmount:
+      title: Fee amount
+      type: number
+      examples:
+        - 5.00
+      description: Amount of the fee that is charged for the transaction; not provided by all Card Service Providers.
+
+    FeeCurrency:
+      title: Fee currency
+      type: string
+      pattern: '^[a-zA-Z0-9]{3}$'
+      examples:
+        - CHF
+      description: Currency that is used for charging the fee, usually the same as the currency of the card account.
+
+    Correlation:
+      type: string
+      examples:
+        - f058ebd6-02f7-4d3f-942e-904344e8cde5
+      description: Schema for X-Correlation-ID header.
+
+    Date:
+      type: string
+      format: date
+      examples:
+        - 2018-04-13
+      description: General date that is used for different attributes, format is 'YYYY-MM-DD'.
+
+    DateTime:
+      title: Date-time
+      type: string
+      format: date-time
```

Comment:
`format: date-time` MUST follow the ISO 8601 respectively RFC 3339. In other words libraires should not have a problem to support also 'YYYY-MM-DDThh:mm:ssZ' or 'YYYY-MM-DDThh:mm:ss+02:00'

Reply:
We are not sure what you mean with the feedback. Is it a general hint or do you expect some changes in the format definition or example?

Reply:
No additional input was received for the feedback so we don't change anything.